budgetportal/models.py

```diff
@@ -2083,3 +2087,137 @@ def csv_url(aggregate_url):
             % URL_LENGTH_LIMIT
         )
     return csv_url
+
+
+class WagtailHomePage(Page):
+    parent_page_types = []
+    subpage_types = ["budgetportal.LearningIndexPage", "budgetportal.PostIndexPage"]
+
+    class Meta:
+        app_label = "budgetportal"
+
+
+class LearningIndexPage(Page):
+    subpage_types = ["budgetportal.GuideIndexPage"]
+    max_count = 1
+
+
+class PostIndexPage(Page):
+    subpage_types = ["budgetportal.PostPage"]
+    max_count = 1
+    intro = RichTextField(blank=True)
+
+    content_panels = Page.content_panels + [FieldPanel("intro", classname="full")]
+
+
+class GuideIndexPage(Page):
+    max_count = 1
+    parent_page_types = ["budgetportal.LearningIndexPage"]
```

Comment:
no need to specify this in both directions - could just use one of parent_page_types or subpage_types

Reply:
I'm not sure its a good aproach, maybe I'm doing something worng.

Reply:
What I mean is its not working fully correctly without both.

Reply:
As you cannot define root types. So under the root you can create all of pages without defining both

Reply:
oh interesting...maybe change since v1. cool, doesn't hurt right now - just thought it could maintenance easier and reduce surprises